tests/template/srw_shadow_test.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+"""PyTest for :mod:`sirepo.template.srw_shadow`
+
+:copyright: Copyright (c) 2020 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from __future__ import absolute_import, division, print_function
+from pykern import pkio
+from pykern import pkunit
+from pykern.pkdebug import pkdc, pkdp, pkdlog, pkdexc
+import pytest
+
+
+def test_conversion():
+    from sirepo.template.srw_shadow import Convert
+
+    for d in pkunit.case_dirs():
```

Comment:
Here's a subtlety about declarative naming. The directories are named to_shadow.in, and to_shadow.out. If the files in them were named in.json and out.json for both cases, you could write the code like this:

```py
pkjson.dump_pretty(
     getattr(Convert(), d.basename)(
         pkjson.load_any("in.json").models,
         "out.json",
     ),
)
```

I didn't want to hold back the review for this, just noting that the code would be a lot shorter.

Reply:
Good point 👍